src/types.tsx

```diff
@@ -107,6 +107,8 @@ export type NavigationStackOptions = HeaderOptions & {
     horizontal?: number;
   };
   disableKeyboardHandling?: boolean;
+  cardStyleInterpolator?: CardStyleInterpolator;
```

Comment:
Are you sure that these `types` shouldn't be added anywhere else?

Reply:
Someone with more experience with those typings should comment on that. I believe that `NavigationStackOptions` will be used in every applicable case of providing `navigationOptions`.

Reply:
It should be enough to add here.

Reply:
If TS check passes, it's all good :)